sdk/Sdk/Tasks/ZipDeploy/CreateZipFileTask.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+﻿using System;
+using System.IO;
+using System.IO.Compression;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+// IMPORTANT: Do not modify this file directly with major changes
+// This file is a copy from this project (with minor updates) -- https://github.com/Azure/azure-functions-vs-build-sdk/blob/b0e54a832a92119e00a2b1796258fcf88e0d6109/src/Microsoft.NET.Sdk.Functions.MSBuild/Microsoft.NET.Sdk.Functions.MSBuild.csproj
+// Please make any changes upstream first.
+
+namespace Microsoft.NET.Sdk.Functions.MSBuild.Tasks
```

Comment:
I kept the namespaces same as what they are in https://github.com/Azure/azure-functions-vs-build-sdk/tree/main/src/Microsoft.NET.Sdk.Functions.MSBuild

I went back and forth on whether to update these to be "Microsoft.Azure.Functions.Worker.Sdk.MSBuild" or not. I thought maybe leave it to indicate that this is a copy (with some changes to get rid of warning like nullables and not using Resources.resx).

I am open to counter arguments here. Really not a big deal either way, as this isn't some exposed API.

Reply:
I'm good with this. The less we diverge, the better at this point.